tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
```

Comment:
Can't comment on `tests/__init__.py` so I'll do it here. Shouldn't the file have a license header?